src/main/java/frc/trigon/robot/subsystems/ledstrip/LEDStripConstants.java

```diff
@@ -0,0 +1,19 @@
+package frc.trigon.robot.subsystems.ledstrip;
+
+import com.ctre.phoenix.led.CANdle;
+import com.ctre.phoenix.led.CANdleConfiguration;
+
+public class LEDStripConstants {
+    private static final int CANDLE_ID = 0;
+    private static final CANdle.LEDStripType STRIP_TYPE = CANdle.LEDStripType.RGB;
+    private static final double BRIGHTNESS_SCALAR = 1;
+    static final CANdle CANDLE = new CANdle(CANDLE_ID);
+
```

Comment:
I haven't really explained this yet, but all of our motors will (hopefully) run on a CANivore. You can read more about it if you'd like. The tl;dr is that a CANivore provides us with a much faster CAN network, and when we create the devices in the code, we need to give them the name of the CANivore. This name is stored in RobotConstants